kafka/admin/client.py

```diff
@@ -473,7 +473,7 @@ def delete_topics(self, topics, timeout_ms=None):
         return response
 
 
-    def _get_cluster_metadata(self, topics=None, auto_topic_creation=False):
```

Comment:
it'd be cleaner if we could specify the broker ID here, and then the caller can pass in a function that identifies the controller... but looks like that's not how `self._send_request_to_controller()` is wired up. And it'd be hard to change because of needing to handle retries. So let's just go with this as-is (especially since it's non-public).